songpal/group.py

```diff
@@ -202,19 +202,19 @@ async def set_codec(self, codectype=0x0040, bitrate=0x0003):
     async def abort(self):
         """Abort current group session."""
         state = await self.state()
-        res = await self.call("X_Abort", MasterSessionID=state.MasterSessionID)
+        res = await self.call("X_Abort", MasterSessionID=state.SessionID)
```

Comment:
Hmm, odd to see this being displayed here. The MasterSessionID -> SessionID change got already merged in, no?

Reply:
Probably my poor git skills. No clue how to fix it either, sorry!